js/pdf_exporter.d.ts

```diff
@@ -1,4 +1,49 @@
-import dxDataGrid from './ui/data_grid';
+import dxDataGrid, { dxDataGridColumn } from './ui/data_grid';
+
+export interface PdfDataGridCell {
```

Comment:
Looks similar to https://js.devexpress.com/Documentation/ApiReference/Common/Object_Structures/ExcelDataGridCell/
Can we make ExportingDataGridCell?

Reply:
Yes, **PdfDataGridCell** and **ExcelDataGridCell** are similar. The only differences are in the prefixes of the names Pdf... Excel...
As an option, we can make a common item **ExportingDataGridCell**.
@EugeniyKiyashko how painless will the renaming be? Can we make it?
Or is it better to make it as a separate task?!
It will also affect the **ExcelPivotGridCell** object in the future.

Reply:
We discussed it with Vladimir earlier. We desided that we remove the dublicate in future and perceive as a separate task. It can take some time and require the help of techwriter for renaming of interfices in docs.

Reply:
Fine